variables.tf

```diff
@@ -0,0 +1,93 @@
+variable "num" {
+  description = "Specify the amount of Windows agents."
+}
+
+variable "cluster_name" {
+  description = "Name of the DC/OS cluster"
+}
+
+variable "location" {
+  description = "Azure Region"
+}
+
+variable "name_prefix" {
+  description = "Name Prefix"
+  default     = ""
+}
+
+variable "vm_size" {
+  description = "Azure virtual machine size"
+}
+
+variable "dcos_instance_os" {
+  description = "Operating system to use. Instead of using your own AMI you could use a provided OS."
+}
+
+variable "image" {
+  description = "Source image to boot from"
+  type        = "map"
+  default     = {}
+}
+
+variable "disk_type" {
+  description = "Disk Type to Leverage"
+  default     = "Standard_LRS"
+}
+
+variable "disk_size" {
+  description = "Disk Size in GB"
+  default     = "120"
+}
+
+variable "resource_group_name" {
+  description = "Name of the azure resource group"
+}
+
+variable "custom_data" {
+  description = "User data to be used on these instances (cloud-init)"
+  default     = ""
+}
+
+variable "admin_username" {
+  description = "Windows admin user"
+  default     = "dcosadmin"
+}
+
+variable "admin_password" {
+  description = "Windows admin password"
+  default     = []
+}
+
+variable "public_ssh_key" {
+  description = "SSH Public Key"
+}
+
+variable "allow_stopping_for_update" {
```

Comment:
not used, use the same defaults for all non windows related values as in https://github.com/dcos-terraform/terraform-azurerm-instance/blob/support/0.2.x/variables.tf